site/templates/application/footer/social-media-buttons.jet

```diff
@@ -0,0 +1,44 @@
+{{block socialMediaButtonsFooter()}}
+  {{social_link_instagram := config("social_link_instagram")}}
+  {{social_link_youtube := config("social_link_youtube")}}
+  {{social_link_facebook := config("social_link_facebook")}}
+  {{social_link_twitter := config("social_link_twitter")}}
+  {{social_link_letterboxd := config("social_link_letterboxd")}}
+  {{if social_link_instagram != "" || social_link_youtube != "" || social_link_facebook != "" || social_link_twitter != "" || social_link_letterboxd != ""}}
+    <div class="social-media-icons">
+    {{if social_link_instagram != ""}}
+      <a target="_blank" title="Instagram" href="{{social_link_instagram}}" class="social-media-icon insta">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
```

Comment:
We are storing these icons [here](https://github.com/shift72/core-template/blob/develop/site/static/fonts/fa-s72.woff) so no need to use hard coded svg.

Reply:
example of usage https://github.com/indiereign/shift72-web/blob/526cc56e9deb36be3de2ceab21d3d96458b886b5/src/ui/modal/share-modal.component.js#L76

Reply:
Only Facebook and Twitter icons in the font file. Have left them all as SVGs to keep a consistent style between them. Thoughts?

Reply:
I think you should move the Instagram/Letterboxd icons into the font file.

Reply:
All logos are in the font file. Bex has signed off on them